src/Symfony/Bundle/FrameworkBundle/Test/ForwardCompatTestTrait.php

```diff
@@ -13,6 +13,8 @@
 
 use PHPUnit\Framework\TestCase;
 
+@trigger_error(sprintf('The "%s" trait is deprecated since Symfony 4.3, typehint methods "setUp" and "tearDown" with ": void" instead.', ForwardCompatTestTrait::class), E_USER_DEPRECATED);
```

Comment:
```suggestion
@trigger_error(sprintf('The "%s" trait is deprecated since Symfony 4.4, typehint methods "setUp" and "tearDown" with ": void" instead.', ForwardCompatTestTrait::class), E_USER_DEPRECATED);
```